### PR TITLE
fix some type issues for geo/consent that only show up on newer CC versions

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

File: extensions/amp-consent/0.1/amp-consent.js [hunk 1]
```diff
@@ -619,7 +624,7 @@ export class AmpConsent extends AMP.BaseElement {
         this.user().error(TAG, 'child element of <amp-consent> with ' +
           `promptUI id ${promptUI} not found`);
       }
-      this.consentUI_[instanceId] = element;
+      this.consentUI_[instanceId] = /** @type {!Element} */ (element);
```
Comment (zhouyx) [inline extensions/amp-consent/0.1/amp-consent.js]: `assertElement` ?

File: extensions/amp-geo/0.1/amp-geo.js [hunk 2]
```diff
@@ -92,6 +92,17 @@ const mode = {
...
+/**
+ * @typedef {{
+ *   ISOCountry: string,
+ *   matchedISOCountryGroups: !Array<string>,
+ *   allISOCountryGroups: !Array<string>,
+ *   isInCountryGroup: GEO_IN_GROUP,
+ *   ISOCountryGroups: !Array<string>
+ * }}
+ */
+export const GeoDef = {};
```
Comment (zhouyx) [inline extensions/amp-geo/0.1/amp-geo.js]: Who is importing the `GeoDef` here?

Comment (jpettitt) [reply extensions/amp-geo/0.1/amp-geo.js]: I'd like to understand that too.

Comment (erwinmombay) [reply extensions/amp-geo/0.1/amp-geo.js]: sorry bout that, I've fixed the references now. GeoDef is the return of `addToBody_` and it is the type you get from the `geoForDocOrNull` service locator

**Comment by erwinmombay** [comment]
@zhouyx @jpettitt PTAL

**Comment by zhouyx** [comment]
Thank you @erwinmombay for the fix!

**Pull request merged**